README.md

```diff
@@ -1,3 +1,86 @@
 # node-sdk-core
-This project contains the core functionality used by Node SDK's generated by the IBM OpenAPI 3 SDK Generator (openapi-sdkgen).
-Code generated by openapi-sdkgen will depend on the function contained in this project.
+This project contains the core functionality used by Node SDK's generated by the IBM OpenAPI 3 SDK Generator (`openapi-sdkgen`).
+Code generated by `openapi-sdkgen` will depend on the functionality contained in this project.
+
+## Installation
+`npm install ibm-cloud-sdk-core`
+
+## Usage
+This package exports a single object containing a number of modules as top level proerties. This object can be imported all at once or specific 
```

Comment:
s/proerties/properties/

Not sure what "all at once or specific" means exactly.